site/app/templates/polls/ViewPoll.twig

```diff
@@ -169,8 +169,8 @@
             [window.getComputedStyle(document.body).getPropertyValue('--histogram-correct'),
                 window.getComputedStyle(document.body).getPropertyValue('--histogram-default')];
             {% for option in poll.getOptions() %}
-                data[0].x.push("{{ option.getResponse()|replace({"\n": " ", "\r": " ", "\t": " "}) }}");
-                data[0].y.push({{ response_counts[option.getId()] }});
+                data[0].x.push("{{ option.getResponse()|raw |escape('js') |replace({"\n": " ", "\r": " ", "\t": " "}) }}");
```

Comment:
Does this work without the `raw`? Can also shorten `escape('js')` to `e('js')`

Reply:
Thanks @cjreed121 for the suggestion.
Yeah, after researching and testing I found we actually don't need `raw` for our application